### PR TITLE
Initial deployment of cron

### PR DESCRIPTION
**Pull request by dzaslavskiy**
This application will allow us to run scheduled jobs in our cf environment.

### CODE REVIEW

File: cron/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Job Scheduler
+
+This application provides the ability to run jobs aon a schedule in the cf environment. It is based on [Supercronic](https://github.com/aptible/supercronic). It has the ability to run Cloud Foundry tasks using the [CF v8 CLI](https://github.com/cloudfoundry/cli) using the [run-task](https://cli.cloudfoundry.org/en-US/v8/run-task.html) command.
```
Comment (folksgl) [inline cron/README.md:3]:
```suggestion
This application provides the ability to run jobs on a schedule in the CF environment. It is based on [Supercronic](https://github.com/aptible/supercronic). It has the ability to run Cloud Foundry Tasks using the [CF v8 CLI](https://github.com/cloudfoundry/cli) using the [run-task](https://cli.cloudfoundry.org/en-US/v8/run-task.html) command.
```

File: cron/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    no-route: true
+    health-check-type: process
+    memory: 64MB
+    instances: 1
+    buildpacks:
+      - https://github.com/cloudfoundry/apt-buildpack
+      - binary_buildpack
+    command: |
+      ./install.sh
+      ./supercronic crontab 2>&1
```
Comment (folksgl) [inline cron/manifest.yaml:13]: Log messages should be going to `stderr` (file descriptor 2), while program output should be sent to `stdout` (file descriptor 1). I would expect that what we'll see most of the time from these jobs is log messages, correct?
```suggestion
      ./supercronic crontab
```

File: cron/crontab [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@hourly echo "Hello, world."
+
+@hourly cf run-task cron --command 'echo "Hello, world."' --name "task"
```
Comment (folksgl) [inline cron/crontab:3]: Assuming these are meant mostly as a kind of sanity check to ensure that the app can run the most basic of jobs and CF Tasks, adding that to the commands will help us identify them later. Even as examples, I think this gets the point across slightly better.
```suggestion
@hourly echo "Testing cron runner"

@hourly cf run-task cron --command 'echo "Test CF Task"' --name "test-cron-task"
```

**Review by folksgl — CHANGES_REQUESTED** [review]

File: cron/manifest.yaml (hunk 2 above)
Comment (dzaslavskiy) [inline cron/manifest.yaml:13]:
```suggestion
      ./supercronic crontab
```

Comment (dzaslavskiy) [reply cron/manifest.yaml]: not needed

**Review by folksgl — APPROVED** [review]

**Pull request merged**